source/CMakeLists.txt

```diff
@@ -49,6 +49,8 @@ if( DQM4HEP_TESTING )
   dqm4hep_add_test( NAME test-xmlparser ARGS "${PROJECT_SOURCE_DIR}/tests/test_xmlparser.xml" )
   
   # Super-custom test
+  set(CUSTOM_TEST_LIBRARY_DIR ${${PROJECT_NAME}_DEPENDS_LIBRARY_DIRS})
```

Comment:
Maaaaaaad world
https://www.youtube.com/watch?v=4N3N1MlvVc4

Reply:
Yeah...took me more than hour to find the problem and make this work